### ISSUE TITLE
Readonly environment

### ISSUE THREAD

**Issue by misuedkaAtQuanos** [mashpie/i18n-node#516]
With a default setup, I experienced write access errors on a readonly enviroment (serverless, Amazon Lambda), I assume i would have to supply the JSON files in the dist folder. But I want to be sure that there will be never any errors from write acesses I cannot look ahead right now.
Wouldn't it be useful? to comment on such a setup on the README page? I searched in this forum and Stackoverflow but couldn't find the final answer.

**Comment by mashpie** [comment]
True, it's a bit hidden in https://github.com/mashpie/i18n-node#list-of-all-configuration-options.

Do you want to provide an appropriate snippet for README or like to submit a PR?

**Comment by misuedkaAtQuanos** [comment]
Of course, I read this list. I'm quite sure that "updateFiles" is a hot candidate. But because I wasn't sure if this is sufficient, I created this ticket. I'm still not sure.
